packages/rpc-subscriptions/src/rpc-subscriptions.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from '@solana/rpc-subscriptions-spec';
 import { ClusterUrl } from '@solana/rpc-types';
 
-import { DEFAULT_RPC_CONFIG } from './rpc-default-config';
+import { DEFAULT_RPC_SUBSCRIPTIONS_CONFIG } from './rpc-default-config';
```

Comment:
I had to make these default configs public so that people could build atop a regular Solana RPC API. Any thoughts on:

1. Whether these belong here?
2. If they should be public or if we should figure something else out?
3. The fact that they're identical copies of each other for both RPC and RPC Subscriptions?

Reply:
@lorisleiva, you last moved these around, right? Any thoughts on if this belongs here, or where it should go?

Reply:
Hey Steven, I'm not seeing anything on the example that justifies making this public. Could you provide a code snippet that shows why we're having to expose this? Regarding the fact that the configs are the same for RPC Subscriptions, I'd still leave them separate as we're bound to have more configs on them in the future that's likely to be specific to RPC or RPC Subscriptions.

Reply:
I need it to be able to create a default Solana RPC API to build new methods on top of.

```ts
createSolanaRpcApi<SolanaRpcApiMainnet>(DEFAULT_RPC_CONFIG);
```

I made the subscriptions one public for the sake of symmetry.

Reply:
Makes sense. 👌